tracer/src/Datadog.Tracer.Native/Datadog.Tracer.Native.DLL.vcxproj

```diff
@@ -53,6 +69,19 @@
     <WholeProgramOptimization>true</WholeProgramOptimization>
     <CharacterSet>Unicode</CharacterSet>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM64' OR '$(Configuration)|$(Platform)'=='Debug|ARM64EC'" Label="Configuration">
+    <ConfigurationType>DynamicLibrary</ConfigurationType>
+    <UseDebugLibraries>true</UseDebugLibraries>
+    <PlatformToolset>v143</PlatformToolset>
```

Comment:
It looks like these require the `v143` toolset. We should consider moving everything to `v143` in a separate change so we only have one toolset we need to install

Reply:
> Yes, but in this case I didn't want to change the current build,

I missed that. If we're going to need that toolset, we will need to update the base images in CI, and we should definitely update to the v143 toolset _first_ in a separate PR, as that will require a bunch of CI config dancing. 

IIRC, there were some issues with toolsets when we first integrated the continuous profiler, but hopefully this doesn't come up here. I can't remember what the issue was, so we'll see 😅

Reply:
https://github.com/DataDog/dd-trace-dotnet/pull/3964